lib/registration.pm

```diff
@@ -153,6 +153,9 @@ sub accept_addons_license {
         addon_decline_license;
         wait_still_screen 2;
         send_key $cmd{next};
+        while (check_screen('import-untrusted-gpg-key', 60)) {
```

Comment:
The function name is accept_addons_license , but you put gpg-key logic here.
maybe we can put logic after the call of accept_addons_license ?

Reply:
Updated